tests/stub/optimizations/scripts/v4x3/pull_pipeline.script

```diff
@@ -4,7 +4,7 @@
 
 {*
     C: RESET
-    S: SUCCESS
+    S: SUCCESS {}
```

Comment:
Is it just for clarity?

Reply:
Nope. It could make drivers fail that expect every `SUCCESS` message to have one field `metadata::Dictionary` which is what the docs say. So the driver would be right in doing so.